apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/CustomIdGenerator.kt

```diff
@@ -0,0 +1,24 @@
+package com.apollographql.apollo.compiler
+
+interface CustomIdGenerator {
+  fun apply(
+      /**
+       * The minified query string source with fragments
+       */
+      queryString: String,
```

Comment:
nit: how about name it as `operationDocument`? As it will be used not only for queries but for subscriptions as well.